### PR TITLE
(WIP) Adding script to apply transform to bvecs

### PR DESCRIPTION
**Pull request by karanphil**
Small script based on the `scil_apply_transform_to_image.py` script. It takes as input a bvecs file and a transformation matrix and applies that matrix on the vectors. Pretty straightforward, but I would like people to test it, I might have missed something. @AlexVCaron @arnaudbore

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-696/2/

File: scripts/scil_apply_transform_to_bvecs.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, [args.in_bvecs, args.in_transfo])
+    assert_outputs_exist(parser, args, args.out_bvecs)
+
+    transfo = load_matrix_in_any_format(args.in_transfo)
```
Comment (AlexVCaron) [inline scripts/scil_apply_transform_to_bvecs.py:47]: You can preselect the rotation only here
```suggestion
    transfo = load_matrix_in_any_format(args.in_transfo)[:3, :3]

```

**Review by AlexVCaron — CHANGES_REQUESTED** [review]

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Review by arnaudbore — APPROVED** [review]

**Pull request merged**